cryptoads/CRBN
# chart data generating from user db data and display CRBN score

implemented a calculateScore() function that takes user db value, builds the FootprintChart based on them and displays the CRBN Score above the chart

TO TEST: 

- update a user to have values for all scores (will need to hard code maintenance and waste values for right now)
- log in with that user
- chart data should populate with db values
